backend/src/main/java/org/sefglobal/core/controller/CertificateController.java

```diff
@@ -0,0 +1,39 @@
+package org.sefglobal.core.controller;
+
+import org.sefglobal.core.exception.ResourceNotFoundException;
+import org.sefglobal.core.model.Certificate;
+import org.sefglobal.core.service.CertificateService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/fellowship")
+public class CertificateController {
+
+    private final CertificateService certificateService;
+
+    public CertificateController(CertificateService certificateService) {
+        this.certificateService = certificateService;
+    }
+
+    @GetMapping("/certificates")
+    public List<Certificate> getCertificates(){
```

Comment:
You can add `@ResponseStatus()` annotations to all the methods to define what will be the response type if method execution passes successfully